### PR TITLE
test: large generated accountdb cluster test

### PR DESCRIPTION
**Pull request by brianolson**
## Summary

Repair features for generating pre-built account database with many accounts already in it (to test sqlite3 account db performance at different sizes).

## Test Plan

This is used in cluster tests.

```
'netgoal' 'build' '-r' '/data/networks/bolson-s3/gen' '-n' 'bolson-s3' '--recipe' 'test/testdata/deployednettemplates/recipes/bootstrappedScenario/recipe.json' '--gen-db-files'
```

### CODE REVIEW

File: netdeploy/remote/deployedNetwork.go [hunk 1]
```diff
@@ -382,10 +385,10 @@ func (cfg DeployedNetwork) GenerateDatabaseFiles(fileCfgs BootstrappedNetwork, g
 		}
 
 		accounts[addr] = alloc.State
-
 	}
 
 	//initial state
+	log := logging.NewLogger()
```
Comment (algorandskiy) [inline netdeploy/remote/deployedNetwork.go:391]: this inits local var, not global one. Maybe move it into init() ?

File: netdeploy/remote/deployedNetwork.go [hunk 2]
```diff
@@ -668,16 +677,17 @@ func createSignedTx(src basics.Address, round basics.Round, params config.Consen
 				sgtxns = append(sgtxns, t)
 			}
 			bootstrappedNet.nAccounts -= uint64(len(sgtxns))
-			bootstrappedNet.accounts = accounts
 			if bootstrappedNet.nAssets > 0 {
+				bootstrappedNet.log.Info("switch to acfg mode")
```
Comment (algorandskiy) [inline netdeploy/remote/deployedNetwork.go:681]: crashes here
```
	/opt/cibuild/.gimme/versions/go1.17.13.linux.amd64/src/runtime/panic.go:1047 +0x266
github.com/algorand/go-algorand/netdeploy/remote.createSignedTx({0xe9, 0x68, 0x52, 0x81, 0xc2, 0x8d, 0x6f, 0x6a, 0x7b, 0x70, ...}, ...)
	/opt/cibuild/project/netdeploy/remote/deployedNetwork.go:681 +0x1eb4
```

**Review by algorandskiy — COMMENTED** [review]
Could you also post command line on how to generate the db and run a test with it?

**Comment by algorandskiy** [comment]
@brianolson please rebase on master to fix the build, and add usage examples into the PR description (what to run cli to have 30M accounts pregenerated before the test)

File: netdeploy/remote/deployedNetwork.go [hunk 3]
```diff
@@ -627,20 +634,22 @@ func accountsNeeded(appsCount uint64, assetCount uint64, params config.Consensus
 func createSignedTx(src basics.Address, round basics.Round, params config.ConsensusParams, bootstrappedNet *netState) ([]transactions.SignedTxn, error) {
 
 	if bootstrappedNet.nApplications == 0 && bootstrappedNet.nAccounts == 0 && bootstrappedNet.nAssets == 0 {
+		if !bootstrappedNet.accountsCreated {
```
[the remark this answers is not in the document]
Comment (brianolson) [reply netdeploy/remote/deployedNetwork.go:637]: This is printed the first nime that nApplications == 0 and nAccounts == 0, nothing more to create, so we're done creating.

**Review by algorandskiy — APPROVED** [review]

**Pull request merged**